cymetric/metrics.py

```diff
@@ -230,6 +230,38 @@ def agents(series):
 del _agentsdeps, _agentsschema
 
 
+# Transaction Masses [kg]
+_transdeps = [
+    ('Materials', ('SimId', 'ResourceId', 'ObjId', 'TimeCreated'), 'Mass'), 
+    ('Transactions', ('SimId', 'TransactionId', 'SenderId', 'ReceiverId', 
+        'ResourceId'), 'Commodity')
+    ]
+
+_transschema = [
+    ('SimId', ts.UUID), ('TransactionId', ts.INT), 
+    ('ResourceId', ts.INT), ('ObjId', ts.INT), 
+    ('TimeCreated', ts.INT), ('SenderId', ts.INT), 
+    ('ReceiverId', ts.INT), ('Commodity', ts.STRING), 
+    ('Mass', ts.DOUBLE)
+    ]
+
+@metric(name='TransactionQuant', depends=_transdeps, schema=_transschema)
```

Comment:
There is not! I welcome any and all input since it looks like this will be a part of the 1.3 release (you can skim metrics.py right now since the PR for online documentation is still waiting on input).

Thank you for the suggestion here, I will make the change.